pkg/apis/credentialprovider/README.md

```diff
@@ -0,0 +1 @@
+Temporarily copied from code that will land in k8s v1.20 - please reach out to SIG Cloud Provider with any questions.
```

Comment:
Apply same treatment from pkg/gcpcredential/README.md to this README too. Link to PR is the most important part.

Remove "please reach out to SIG" text. If we want a channel for communication, create a issue for tracking and reference it instead.

Reply:
Added the PR link - it sounded like we wanted to keep the SIG text in after discussing that in chat, so I left it in for now, but I can definitely change that if need be.